shell/initialize/client.js

```diff
@@ -93,6 +106,8 @@ if (debug) {
           const currentApp = vm.$root[nuxtApp];
 
           // Load error layout
+          // why is NuxtError being exported on /initialize/index.js when it's just being imported there?
+          // we can import it here and remove from there!
```

Comment:
I'd say it's a somewhat common usage. Libraries will often define things in one file and then just import and export in another file to improve the usability for consumers. 

There's even some syntactic sugar to support it.
```
export { encrypt as en } from 'crypto'; 
export * from 'crypto';                
```